.github/workflows/compile-mermaid.yml

```diff
@@ -26,13 +26,17 @@ jobs:
 
     - name: Docker build
       run: docker build -t ${{env.IMAGETAG}} . -f $DOCKERFILE
- 
+
     - name: Test if the CLI actually works (PNG)
-      run: for i in $(ls $INPUT_DATA/*.mmd); do docker run -v $(pwd):/data ${{env.IMAGETAG}} -i /data/$i -o /data/$i.png -w 800; done
-    
+      run: |
+        for i in $(ls $INPUT_DATA/*.mmd); do docker run -v $(pwd):/data ${{env.IMAGETAG}} -i /data/$i -o /data/$i.png -w 800; done
+        for i in $(ls $INPUT_DATA/*.mmd); do cat $i | docker run -i -v $(pwd):/data ${{env.IMAGETAG}} -o /data/$i-stdin.png -w 800; done
```

Comment:
To my understanding `-i` is not needed here anymore. If that is the case, please, update both workflow files.

Reply:
The `-i` option is being passed into docker, not mermaid-cli. It tells the docker container to run an interactive session, which allows us to `cat` a file and pipe it into docker. Otherwise, docker will ignore any stdin piped into it from the command line.